### ISSUE TITLE
Contacts must be able to be deleted

### ISSUE THREAD

**Issue by bradleybeddoes** [ausaccessfed/federationregistry#240]
Currently this is a manual process and some universities want to clean these up from time to time.

Also linking together `Contact` and `Subject` at the table level was a terrible idea by @bradleybeddoes.

![self five](https://media.giphy.com/media/cJgkhpLgsuTkY/giphy.gif)

**Comment by vladimir-mencl-eresearch** [comment]
Hi @bradleybeddoes , just going through backlog of tickets - can this one be closed as resolved by #243 ?

**Comment by bradleybeddoes** [comment]
I don't think so, this is about being able to delete, the referenced ticket just allows deactivation.

**Comment by vladimir-mencl-eresearch** [comment]
I thought deactivation was considered as sufficient - thanks for the clarification.
